R/models.R

```diff
@@ -103,14 +104,20 @@ cloudml_predict <- function(
 
   gcloud <- gcloud_config(gcloud)
 
-  json_file <- tempfile(fileext = ".json")
-  jsonlite::write_json(input, json_file)
+  # CloudML CLI does not expect valid JSON but rather a one line per JSON instance.
+  # See https://cloud.google.com/ml-engine/docs/online-predict#formatting_your_input_for_online_prediction
```

Comment:
Believe it or not this is actually a thing (https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON). The R jsonlite package actually supports this directly (https://www.opencpu.org/posts/jsonlite-streaming/)

Reply:
The `stream_out` function appears to support only data frames so not necessarily useful for this particular case: https://www.rdocumentation.org/packages/jsonlite/versions/1.5/topics/stream_in%2C%20stream_out

Reply:
Interesting!

Right, looks like `stream_out` is optimized for data frames for now.